src/analyses/libraryDesc.ml

```diff
@@ -38,13 +38,16 @@ type special =
   | Assert of { exp: Cil.exp; check: bool; refine: bool; }
   | Lock of { lock: Cil.exp; try_: bool; write: bool; return_on_success: bool; }
   | Unlock of Cil.exp
+  | CondWait of { cond: Cil.exp; lock: Cil.exp }
   | ThreadCreate of { thread: Cil.exp; start_routine: Cil.exp; arg: Cil.exp; }
   | ThreadJoin of { thread: Cil.exp; ret_var: Cil.exp; }
   | ThreadExit of { ret_val: Cil.exp; }
   | Math of { fun_args: math; }
   | Memset of { dest: Cil.exp; ch: Cil.exp; count: Cil.exp; }
   | Bzero of { dest: Cil.exp; count: Cil.exp; }
+  | Memcpy of { dest: Cil.exp; src: Cil.exp } (* TODO: separate for strcpy, etc *)
```

Comment:
I'll add `Strcpy`, but there's no point in adding fields for data we don't actually use. This way it's clear that we don't, but otherwise it might look like we do. Only by looking at the `special` implementations of specific analyses, it would become apparent that we actually don't use it at all.
It's trivial to add it if we decide to do something with it.